src/objective-c/GRPCClient/internal_testing/GRPCCall+Tests.h

```diff
@@ -45,6 +45,9 @@
  *
  * Must be called before any gRPC call to that host is made. It's illegal to pass the same host to
  * more than one invocation of the methods of this category.
+ *
+ * This function is for internal testing of gRPC only. It is not part of gRPC's public interface.
+ * Do not use in production. To enable, set the preprocessor flag GRPC_TEST_OBJC.
```

Comment:
No, not these! :D These are for users to test their apps, not only for us. These are already part of the public interface too: even if we wanted to hide them the only way would be to deprecate them and keep them until 2.0.